.travis.yml

```diff
@@ -5,6 +5,7 @@ python:
 before_install:
   - sudo apt-get -qq update
   - sudo apt-get install -y unrtf antiword poppler-utils libjpeg-dev
+  - sudo pip2 install --pre pyhwp
```

Comment:
Since this is a Python dependency, can you place it in `requirements.txt` and remove it from here? This will cause it to be added to the package (`setup.py`) as well as installing it during Travis CI testing.

Reply:
pyhwp doesn't support Py3 yet, And it is planed for run as CLI tool not as Python module. So I believe it has to be treated CLI tool.

Reply:
I don't know how to selective install by Python version and add --pre option to setup.py. Would you mind do that for me?

Reply:
You can do this by adding `pyhwp --pre` to `requirements.txt`.